Lib/turtle.py

```diff
@@ -3402,7 +3444,6 @@ def begin_fill(self):
             self.undobuffer.push(("beginfill", self._fillitem))
         self._update()
 
-
```

Comment:
Can you restore this?

Reply:
Let's keep the change, it's only a bit of whitespace: "this is also an opportunity to clean up someone else’s mess (in true XP style)".